notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -314,6 +405,12 @@
     "    add[\"Z\"], add[\"A\"] = get_Z_A(add[\"Radionuclide\"])\n",
     "    if add[\"Radionuclide\"][-1].isalpha():\n",
     "        add[\"Metastable_state\"] = add[\"Radionuclide\"][-1]\n",
+    "    if (line[0].endswith(\"m\") == True\n",
+    "        or line[0].endswith(\"n\") == True):\n",
+    "        add[\"Atomic_mass\"] = (line[28] +\n",
+    "                              nubase_df.loc[add[\"Radionuclide\"]].Isomer_excitation_energy)\n",
+    "    else:\n",
+    "        add[\"Atomic_mass\"] = line[28]\n",
     "    add[\"Half_life\"] = line[1]\n",
```

Comment:
For this block, it looks like you're using the atomic mass data from ICRP-07 rather than AME2020. For the stable we have to use AME2020, so it would be nicer to be consistent and switch so all the amu data comes from AME2020.